Cython/Utility/NumpyImportArray.c

```diff
@@ -26,7 +26,7 @@ if (unlikely(_import_array() == -1)) {
 
 {
     // NO_IMPORT_UFUNC is Numpy's mechanism for indicating that import_umath is handled elsewhere
-#if !NO_IMPORT_UFUNC /* https://numpy.org/devdocs/reference/c-api/ufunc.html#c.NO_IMPORT_UFUNC */
+#ifndef(NO_IMPORT_UFUNC) /* https://numpy.org/doc/stable/reference/c-api/ufunc.html#c.NO_IMPORT_UFUNC */
```

Comment:
```suggestion
#ifndef NO_IMPORT_UFUNC /* https://numpy.org/doc/stable/reference/c-api/ufunc.html#c.NO_IMPORT_UFUNC */
```

I don't think you want the brackets with `ifndef`